components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -1363,6 +1328,18 @@ else if (getZCTCoords(plane)[2] < timestamps.size()) {
     segments = null;
   }
 
+  // -- ZeissCZI-specific methods --
+
+  public boolean allowAutostitching() {
+    return ((DefaultMetadataOptions) getMetadataOptions()).getBoolean(
```

Comment:
For me these are both breaking changes and generally dangerous. _Minimally_ these need to use an `instanceof`; otherwise these methods would throw if anyone has created their own implementation of the `MetadataOptions` class. See review comments for further thoughts.